code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -814,7 +814,8 @@
 
 /obj/item/weapon/rig/dropped(var/mob/user)
```

Comment:
Should probably find out where this is being called without a `user`; the rest of this proc (both directly and through `toggle_piece`) is dependent on there being a user argument passed.